daemon/guestos_mgr.c

```diff
@@ -297,7 +297,11 @@ guestos_mgr_update_images(void)
 static char *
 write_to_tmpfile_new(unsigned char *buf, size_t buflen)
 {
-	char *file = mem_strdup("/tmp/tmpXXXXXXXX");
+	/* this fixes the problem that /tmp is unshared between cmld and scd which is why
+	 * it cannot be used to share files to be hashed (e.g. newca_certs)
+	 * FIXME: move SCD in own container and introduce shared mount
+	 */
+	char *file = mem_strdup("/data/cml/shared/tmpXXXXXXXX");
```

Comment:
use macro DEFAULT_BASE_PATH instead of hard coding "/data/cml" which brakes LSB builds:
`char *file = mem_printf("%s/shared/tmpXXXXXXXX", DEFAULT_BASE_PATH);`

Reply:
Fixed as per your suggestion  (f2440dbbb5985d87be6b93fe313fef00372df463)